src/ray/gcs/store_client/redis_store_client.cc

```diff
@@ -141,17 +113,56 @@ RedisStoreClient::RedisStoreClient(std::shared_ptr<RedisClient> redis_client)
   RAY_CHECK(!absl::StrContains(external_storage_namespace_, kClusterSeparator))
       << "Storage namespace (" << external_storage_namespace_ << ") shouldn't contain "
       << kClusterSeparator << ".";
+  const int TABLE_COUNT = 8;
+  std::string tables[TABLE_COUNT] = {
```

Comment:
make this a constexpr static variable. Also make a function to do the initial cache population